lib/licenses.nix

```diff
@@ -653,6 +653,11 @@ in mkLicense lset) ({
     fullName = "Non-Profit Open Software License 3.0";
   };
 
+  nvidia-warp = {
+    fullName = "NVIDIA Source Code License for Warp";
+    url = "https://github.com/NVIDIA/warp/blob/main/LICENSE.md";
```

Comment:
This doesn't look to be a free license:

> 3.3 Use Limitation. The Work and any derivative works thereof only may be used or intended for use non-commercially. Notwithstanding the foregoing, NVIDIA and its affiliates may use the Work and any derivative works commercially. As used herein, “non-commercially” means for research or evaluation purposes only.

It looks like it can be redistributable, but not sure.

Reply:
Maybe it would be better to not introduce a new license at all and just use either `licenses.unfree` or `licenses.unfreeRedistributable` instead.